scripts/ci/run_test.sh

```diff
@@ -1,40 +1,5 @@
 #!/usr/bin/env bash
 
 
-# This is a temporary fix to correct the KUBECONFIG file that gets mounted.
-# When the KUBECONFIG file is created by kind, it points to localhost. When inside
-# the cluster, we need to set this value to be .spec.clusterIP instead
-# See: https://github.com/operator-framework/operator-sdk/issues/2618
-KUBERNETES_SERVICE_HOST="$(kubectl get service kubernetes -o jsonpath='{.spec.clusterIP }')"
-temp=$(mktemp)
-cat "${KUBECONFIG}" | sed "s/server: https.*/server: https:\/\/${KUBERNETES_SERVICE_HOST}/g" > "${temp}"
-contents=$(<"${temp}")
-kubectl create cm kube-config --from-literal=kubeconfig="${contents}"
-rm "${temp}"
-
-# create roles and service account required for the test runner
-kubectl apply -f deploy/testrunner
-
-# start the test runner pod
-kubectl run test-runner --generator=run-pod/v1 \
-  --restart=Never \
-  --image-pull-policy=Always \
-  "--image=quay.io/mongodb/community-operator-testrunner:${version_id:?}" \
-  --serviceaccount=test-runner \
-  --command -- \
-  ./runner --operatorImage "quay.io/mongodb/community-operator-dev:${version_id}" \
-           --preHookImage "quay.io/mongodb/community-operator-pre-stop-hook:${version_id}" \
-           --testImage "quay.io/mongodb/community-operator-e2e:${version_id}" \
-           "--test=${test:?}"
-
-
-echo "Test pod is ready to begin"
-kubectl wait --for=condition=Ready pod -l run=test-runner --timeout=600s
-
-# The test will have fully finished when tailing logs finishes
-kubectl logs -f -l run=test-runner
-
-result="$(kubectl get pod -l run=test-runner -o jsonpath='{ .items[0].status.phase }')"
-if [[ ${result} != "Succeeded" ]]; then
-  exit 1
-fi
+pip3 install -r ./requirements.txt
```

Comment:
This is just a suggestion/idea, no need to do this now 

This script can be replaced with a direct call from Evergreen with the form:

```yaml
  run_e2e_test:
    - command: subprocess.exec
      type: test
      params:
        add_to_path:
          - ${workdir}/bin
        working_dir: mongodb-kubernetes-operator
        command: "python3 ./scripts/dev/e2e.py --skip-operator-install 1 --skip-image-build 1 --test ${test} --tag ${version_id} --config_file ./scripts/ci/config.json --dump_diagnostic 1"
        env:
          KUBECONFIG: ${workdir}/kube_config
```

And if we add defaults to `skip-operator-install`, `skip-image-build` and `dump_diagnostic` we could make it even nicer.

Reply:
I like it! thanks!